prometheus/collectors/gen_go_collector_set.go

```diff
@@ -81,14 +81,35 @@ func main() {
 	descriptions := computeMetricsList()
 	groupedMetrics := groupMetrics(descriptions)
 
+	allDesc := metrics.All()
+
+	// Find default metrics.
+	var defaultRuntimeDesc []metrics.Description
+	for _, d := range allDesc {
+		if !internal.GoCollectorDefaultRuntimeMetrics.MatchString(d.Name) {
+			continue
+		}
+		defaultRuntimeDesc = append(defaultRuntimeDesc, d)
+	}
+
+	var defaultRuntimeMetricsList []string
+
+	for _, d := range defaultRuntimeDesc {
+		if trans := rm2prom(d); trans != "" {
```

Comment:
What does `trans` means? Not sure I understand the variable name 🥲

Reply:
Hi, I was still simplifying a bit as announced ;), so I condensed the piece of code you are referring to. Nonetheless, to answer your question. I maintained the naming of that `trans` variable to be consistent with similar pre-existing usages in this file and in `gen_go_collector_metrics_set.go`. Under the hood, the function `rm2prom` calls `internal.RuntimeMetricsToProm` and there I could assume that `trans` stands for "translation".